### DISCUSSION TITLE
Add background image to the MediaElement

### DISCUSSION THREAD

**Discussion by yureng** [CommunityToolkit/Maui#1283] [New Feature Discussions]
Is it possible to add a BackgroundImage property for the MediaElement , when the video Source is not set, it can display this BackgroundImage , not just the BackgroundColor?

Many thanks to the CommunityToolkit team!

**Comment by bijington** [comment]
I would be surprised if the platforms made it possible for this. @jfversluis do you know if something like this would be possible?

**Reply by jfversluis** [reply]
Hm yeah I don't think this is something that would be supported by the platforms so _if_ we were to do this, we'd have to come up with something ourselves. I guess it should be relatively easy to do yourself with some `DataTrigger` or something that checks if a source is set and if not hide the `MediaElement` and show some `Image` in the same place.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
